client/ayon_houdini/hooks/set_paths.py

```diff
@@ -6,6 +6,7 @@ class SetPath(PreLaunchHook):
 
     Hook `GlobalHostDataHook` must be executed before this hook.
     """
+
```

Comment:
I know, cosmetics - but can you revert this too. So that less files are changed.